src/quacc/recipes/onetep/_base.py

```diff
@@ -0,0 +1,77 @@
+"""Base jobs for Onetep."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase import Atoms
+
+### Do we need to import from quacc at this point we can do it directly from ase
+from quacc.calculators.onetep.onetep import Onetep, OnetepProfile
+from ase.calculators.onetep import OnetepTemplate
+from quacc.runners.ase import run_calc
+from quacc.schemas.ase import summarize_run
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from quacc.schemas._aliases.ase import RunSchema
+
+
+def base_fn(
```

Comment:
A lot of these parameters are not relevant for ONETEP and are an artifact of using the custom Espresso calculator as a starting point. I would recommend taking a look at recipes like those for tblite, Gaussian or ORCA for a point of comparison.